### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by andyoe0102**
-dockerize app
-have schema.sql run when docker-composing to create mysql database service

@krisdo @cmonez @samasastudio

### CODE REVIEW

File: MYSQL/tables/schema.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  placeName VARCHAR(255),
+  price DECIMAL(10,2),
+  pictureURL VARCHAR(255)
+);
+
+-- CREATE TABLE roomtype(
+--   id INT(20) PRIMARY KEY NOT NULL,
+--   type VARCHAR(255)
+-- );
+
```
Comment (samasastudio) [inline MYSQL/tables/schema.sql:20]: what brought you to remove roomtype?

File: client/src/components/App/app copy.jsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        <Button onClick={this.toggleModal}>{`Show all ${this.state.reviews.length} reviews`}</Button>
+
+        </Appcontainer>
+
+    )
+  }
+}
+
+export default App;
+
```
Comment (samasastudio) [inline client/src/components/App/app copy.jsx:148]: did you have to make a copy of your app while dockerizing?

File: client/src/components/App/app.jsx [hunk 3]
```diff
@@ -27,6 +31,32 @@ const Button = styled.button`
...
+//   transform:translateY(0):
+// }
+// &.modal-transition-exit-active{
+//   transition:transform 900ms ease-in-out ;
+//   transform: translateY(-100%);
+// }
+
+//  &.modal-transition-appear
+
+// `;
```
Comment (samasastudio) [inline client/src/components/App/app.jsx:54]: why are these commented out?

File: MYSQL/tables/schema.sql (hunk 1 above)
Comment (andyoe0102) [reply MYSQL/tables/schema.sql:20]: This was for Suggestions app service. I am getting roomtype information from Eric. Pretty much all of the information
for suggestions I'll be making api request to eric and Krissy to get the data.

File: client/src/components/App/app.jsx (hunk 3 above)
Comment (andyoe0102) [reply client/src/components/App/app.jsx:54]: I was working on adding animation for my modal but I could not make it to work.  For right now I just commented it out and will go back to try to make it work

**Pull request merged**